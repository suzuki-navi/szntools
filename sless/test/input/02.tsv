code	name	kana
010006	北海道	ﾎｯｶｲﾄﾞｳ
020001	青森県	ｱｵﾓﾘｹﾝ
030007	岩手県	ｲﾜﾃｹﾝ
040002	宮城県	ﾐﾔｷﾞｹﾝ
050008	秋田県	ｱｷﾀｹﾝ
060003	山形県	ﾔﾏｶﾞﾀｹﾝ
070009	福島県	ﾌｸｼﾏｹﾝ
080004	茨城県	ｲﾊﾞﾗｷｹﾝ
090000	栃木県	ﾄﾁｷﾞｹﾝ
100005	群馬県	ｸﾞﾝﾏｹﾝ
110001	埼玉県	ｻｲﾀﾏｹﾝ
120006	千葉県	ﾁﾊﾞｹﾝ
130001	東京都	ﾄｳｷｮｳﾄ
140007	神奈川県	ｶﾅｶﾞﾜｹﾝ
150002	新潟県	ﾆｲｶﾞﾀｹﾝ
160008	富山県	ﾄﾔﾏｹﾝ
170003	石川県	ｲｼｶﾜｹﾝ
180009	福井県	ﾌｸｲｹﾝ
190004	山梨県	ﾔﾏﾅｼｹﾝ
200000	長野県	ﾅｶﾞﾉｹﾝ
210005	岐阜県	ｷﾞﾌｹﾝ
220001	静岡県	ｼｽﾞｵｶｹﾝ
230006	愛知県	ｱｲﾁｹﾝ
240001	三重県	ﾐｴｹﾝ
250007	滋賀県	ｼｶﾞｹﾝ
260002	京都府	ｷｮｳﾄﾌ
270008	大阪府	ｵｵｻｶﾌ
280003	兵庫県	ﾋｮｳｺﾞｹﾝ
290009	奈良県	ﾅﾗｹﾝ
300004	和歌山県	ﾜｶﾔﾏｹﾝ
310000	鳥取県	ﾄｯﾄﾘｹﾝ
320005	島根県	ｼﾏﾈｹﾝ
330001	岡山県	ｵｶﾔﾏｹﾝ
340006	広島県	ﾋﾛｼﾏｹﾝ
350001	山口県	ﾔﾏｸﾞﾁｹﾝ
360007	徳島県	ﾄｸｼﾏｹﾝ
370002	香川県	ｶｶﾞﾜｹﾝ
380008	愛媛県	ｴﾋﾒｹﾝ
390003	高知県	ｺｳﾁｹﾝ
400009	福岡県	ﾌｸｵｶｹﾝ
410004	佐賀県	ｻｶﾞｹﾝ
420000	長崎県	ﾅｶﾞｻｷｹﾝ
430005	熊本県	ｸﾏﾓﾄｹﾝ
440001	大分県	ｵｵｲﾀｹﾝ
450006	宮崎県	ﾐﾔｻﾞｷｹﾝ
460001	鹿児島県	ｶｺﾞｼﾏｹﾝ
470007	沖縄県	ｵｷﾅﾜｹﾝ
