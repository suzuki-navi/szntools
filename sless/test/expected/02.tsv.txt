format=tsv
    │ 1      │ 2        │ 3       
    │ code   │ name     │ kana    
 1  │ 010006 │ 北海道   │ ﾎｯｶｲﾄﾞｳ 
 2  │ 020001 │ 青森県   │ ｱｵﾓﾘｹﾝ  
 3  │ 030007 │ 岩手県   │ ｲﾜﾃｹﾝ   
 4  │ 040002 │ 宮城県   │ ﾐﾔｷﾞｹﾝ  
 5  │ 050008 │ 秋田県   │ ｱｷﾀｹﾝ   
 6  │ 060003 │ 山形県   │ ﾔﾏｶﾞﾀｹﾝ 
 7  │ 070009 │ 福島県   │ ﾌｸｼﾏｹﾝ  
 8  │ 080004 │ 茨城県   │ ｲﾊﾞﾗｷｹﾝ 
 9  │ 090000 │ 栃木県   │ ﾄﾁｷﾞｹﾝ  
 10 │ 100005 │ 群馬県   │ ｸﾞﾝﾏｹﾝ  
 11 │ 110001 │ 埼玉県   │ ｻｲﾀﾏｹﾝ  
 12 │ 120006 │ 千葉県   │ ﾁﾊﾞｹﾝ   
 13 │ 130001 │ 東京都   │ ﾄｳｷｮｳﾄ  
 14 │ 140007 │ 神奈川県 │ ｶﾅｶﾞﾜｹﾝ 
 15 │ 150002 │ 新潟県   │ ﾆｲｶﾞﾀｹﾝ 
 16 │ 160008 │ 富山県   │ ﾄﾔﾏｹﾝ   
 17 │ 170003 │ 石川県   │ ｲｼｶﾜｹﾝ  
 18 │ 180009 │ 福井県   │ ﾌｸｲｹﾝ   
 19 │ 190004 │ 山梨県   │ ﾔﾏﾅｼｹﾝ  
 20 │ 200000 │ 長野県   │ ﾅｶﾞﾉｹﾝ  
 21 │ 210005 │ 岐阜県   │ ｷﾞﾌｹﾝ   
 22 │ 220001 │ 静岡県   │ ｼｽﾞｵｶｹﾝ 
 23 │ 230006 │ 愛知県   │ ｱｲﾁｹﾝ   
 24 │ 240001 │ 三重県   │ ﾐｴｹﾝ    
 25 │ 250007 │ 滋賀県   │ ｼｶﾞｹﾝ   
 26 │ 260002 │ 京都府   │ ｷｮｳﾄﾌ   
 27 │ 270008 │ 大阪府   │ ｵｵｻｶﾌ   
 28 │ 280003 │ 兵庫県   │ ﾋｮｳｺﾞｹﾝ 
 29 │ 290009 │ 奈良県   │ ﾅﾗｹﾝ    
 30 │ 300004 │ 和歌山県 │ ﾜｶﾔﾏｹﾝ  
 31 │ 310000 │ 鳥取県   │ ﾄｯﾄﾘｹﾝ  
 32 │ 320005 │ 島根県   │ ｼﾏﾈｹﾝ   
 33 │ 330001 │ 岡山県   │ ｵｶﾔﾏｹﾝ  
 34 │ 340006 │ 広島県   │ ﾋﾛｼﾏｹﾝ  
 35 │ 350001 │ 山口県   │ ﾔﾏｸﾞﾁｹﾝ 
 36 │ 360007 │ 徳島県   │ ﾄｸｼﾏｹﾝ  
 37 │ 370002 │ 香川県   │ ｶｶﾞﾜｹﾝ  
 38 │ 380008 │ 愛媛県   │ ｴﾋﾒｹﾝ   
 39 │ 390003 │ 高知県   │ ｺｳﾁｹﾝ   
 40 │ 400009 │ 福岡県   │ ﾌｸｵｶｹﾝ  
 41 │ 410004 │ 佐賀県   │ ｻｶﾞｹﾝ   
 42 │ 420000 │ 長崎県   │ ﾅｶﾞｻｷｹﾝ 
 43 │ 430005 │ 熊本県   │ ｸﾏﾓﾄｹﾝ  
 44 │ 440001 │ 大分県   │ ｵｵｲﾀｹﾝ  
 45 │ 450006 │ 宮崎県   │ ﾐﾔｻﾞｷｹﾝ 
 46 │ 460001 │ 鹿児島県 │ ｶｺﾞｼﾏｹﾝ 
 47 │ 470007 │ 沖縄県   │ ｵｷﾅﾜｹﾝ  
